src/background_base.cc

```diff
@@ -298,7 +298,7 @@ void BackgroundBase::NumericalDerivatives(void)
    _spdata._mask >>= mask_offset;
    if(BITS_RAISED(_spdata._mask, BACKGROUND_ALL)) {
 // Derivatives are only needed for trajectory types whose transport assumes the background changes on scales longer than the gyro-frequency.
-      w_g = CyclotronFrequency(Vel(_mom[0]), _spdata.Bmag, specie);
+      w_g = fmin(CyclotronFrequency(Vel(_mom[0]), _spdata.Bmag, specie), _spdata.dmax / Vel(_mom[0]));
```

Comment:
It seems this statement compares a frequency and a time interval.

Reply:
You're right. The second argument in fmin should be Vel / dmax. I'll fix that.